lib/taskana-core/src/main/java/pro/taskana/workbasket/internal/WorkbasketQueryMapper.java

```diff
@@ -31,7 +31,7 @@ public interface WorkbasketQueryMapper {
           + "MAX(PERM_CUSTOM_8::int) as MAX_CUSTOM_8, MAX(PERM_CUSTOM_9::int) as MAX_CUSTOM_9, MAX(PERM_CUSTOM_10::int) as MAX_CUSTOM_10, MAX(PERM_CUSTOM_11::int) as MAX_CUSTOM_11, MAX(PERM_CUSTOM_12::int) as MAX_CUSTOM_12 "
           + "</otherwise>"
           + "</choose>"
-          + "FROM WORKBASKET_ACCESS_LIST where ACCESS_ID IN (<if test='accessId != null'><foreach item='item' collection='accessId' separator=',' >#{item}</foreach></if>) group by WORKBASKET_ID ) a "
+          + "FROM WORKBASKET_ACCESS_LIST where LOWER(ACCESS_ID) IN (<if test='accessIds != null'><foreach item='item' collection='accessIds' separator=',' >#{item}</foreach></if>) group by WORKBASKET_ID ) a "
```

Comment:
This is a "dirty fix" of the underlying issue: the input accessIds are not lowercase.